beacon_node/network/src/sync/manager.rs

```diff
@@ -326,6 +362,9 @@ impl<T: BeaconChainTypes> SyncManager<T> {
             RequestId::SingleBlob { id } => {
                 self.on_single_blob_response(id, peer_id, RpcEvent::RPCError(error))
             }
+            RequestId::DataColumnsByRoot(id) => {
```

Comment:
Perhaps rename this to `SingleDataColumn` for consistency?

Reply:
So `SingleBlob` blob does not really mean 1 blob, it means a blob for a SingleLookup. Here data columns are not strictly part of a SingleLookup as range sync will also consume then. I can do a PR latter to align `SingleBlob` into `BlobByRoot` which is also correct, but using `SingleDataColumn` would be a missnomer.